backfill_corrections/Makefile

```diff
@@ -1,8 +1,19 @@
 SHELL:=/bin/bash
 
-TODAY:=$(shell date -u +"%Y-%m-%d")
-CURR_TIME:=$(shell date -u +"%Hh%Mm%Ss")
-LOG_FILE:=$(TODAY)_$(CURR_TIME).log
+DOCKER_IMAGE=ghcr.io/cmu-delphi/covidcast-indicators-backfill_corrections
+DOCKER_TAG=latest
+# Change training options during `make` call via `make <command> OPTIONS="<options>"`
+OPTIONS=--train_models --make_predictions
```

Comment:
pros/cons of making the default the lengthiest and rarest configuration? it does serve as sortof documentation of the available options but are there other reasons to do it?

Reply:
There's no particular reason for doing this beyond it being convenient for manual testing.

Since we're going to be separating the training and prediction steps in production, we should always modify the `OPTIONS` to be explicit about what procedure we're intending to run (+ robustness in case default `OPTIONS` changes). So the default shouldn't have an impact there.

RE documenting options, the `run.R` file shows them pretty clearly so I wouldn't worry about that too much either way.

Reply:
mmm if we want to always specify OPTIONS explicitly, would it be better to set it empty here and fail if it hasn't been overridden?

Reply:
Sure, that sounds good. The pipeline is already set up to [short-circuit](https://github.com/cmu-delphi/covidcast-indicators/blob/ace7a987c8bdbea144db1ee84dc4dc263cd98c24/backfill_corrections/delphiBackfillCorrection/R/main.R#L232-L235) if no options have been set, so I won't add any extra handling here.